regparser/notice/build.py

```diff
@@ -320,6 +320,11 @@ def process_amendments(notice, notice_xml):
             create_xmlless_changes(other_labels, notice_changes)
 
             for cfr_part, rel_labels in labels_by_part.iteritems():
+            # labels_for_part = {part: labels
+            #                    for part, labels in labels_by_part.iteritems()
+            #                    if part == default_cfr_part}
+            #
+            # for cfr_part, rel_labels in labels_for_part.iteritems():
```

Comment:
Did you mean to include these changes to only parse for the specific regulation we're concerned with? 

If so, this is a breaking change for output that goes to reg-core for reg-site. And I'm also concerned about what it'll mean for when we get to generating notice xml as well (which will necessarily have to be transformed in a single notice JSON for however many regulations a notice applies to).